WORKING-GROUPS.md

```diff
@@ -155,7 +155,7 @@ Build, test, release
 Artifact | Link
 ---|---
 Forum | [istio-test-release@](https://groups.google.com/forum/#!forum/istio-test-release)
-Community Meeting VC | [Thursdays at 13:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
+Community Meeting VC | [Monday at 10:00 PT - Weekly](https://plus.google.com/hangouts/_/google.com/istio-test)
```

Comment:
isn't it at 11a pst ? (10 is the biweekly leads one?)

Reply:
Thanks